docs/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Next
+* For ratings, platform selection has been updated for Shadowkeep - check the setting page to update your selection.
```

Comment:
The script that handles the changelog wants you to have a blank line under the header.